apps/fxc-front/src/app/components/2d/path-element.ts

```diff
@@ -254,6 +250,63 @@ export class PathElement extends connect(store)(LitElement) {
     this.postScoreToHost(score);
   }
 
+  private computeScore(points: LatLon[]): Score {
+    const track: ScoringTrack = {
+      points: points.map((point, i) => {
+        return {
+          ...point,
+          alt: 0,
+          timeSec: i * 60,
+        };
+      }),
+      minTimeSec: new Date().getTime() / 1000,
+    };
+    const result = optimize({ track }, this.getLeague()).next().value;
+    const score = new Score({
```

Comment:
feel free to rename`distance` to `distanceM` or `lengthM`, same for `Closing radius`

Reply:
ok. I renamed the properties